src/main/java/org/klodnicki/service/ReturnBookService.java

```diff
@@ -29,12 +36,34 @@ public void returnBook(String firstName, String lastName, String pesel, String t
         Account account = accountService.findAccountByFirstNameAndLastNameAndPesel(firstName, lastName, pesel);
         BookInfo bookInfo = bookService.findBookByTitleAndAuthorAndEdition(title, author, edition);
 
-        bookInfo.removeAccount(account);
+        LendingInformation lendingInformation = lendingInformationService.findLendingInformationByAccountAndBookInfo
+                (account, bookInfo);
+
+        if (countFineForKeepingBook(lendingInformation) > 0) {
+            System.out.println(FINE_TO_BE_PAID + countFineForKeepingBook(lendingInformation));
```

Comment:
Service using sout? Unacceptable if we have already a view class that was and is responsible for such action.

Reply:
OK! done! -> I changed that into the exception